### ISSUE TITLE
Localhost:5000 Oops Something Went Wrong

### ISSUE THREAD

**Issue by BladeBorn345** [istornz/iPokeGo#317]
Went to bed with my iPokeGo working just fine, woke up the next morning to absolutely nothing pulling on my app. So i checking in on the localhost:5000 and got Oops Something Went Wrong. I haven't been able to find out where anything could have gone wrong especially since it was working just fine for me the night before. I've checked my port 5000 and it is indeed still open and my ip address is reserved so it shouldn't change. HELP ME PLZ!!!

**Comment by cyan** [comment]
i don't think you can use le "localhost:5000" on ipokego, since you would need to have a pogom or pokemap on your phone, and i don't see the need to have ipokego then :/

**Comment by BladeBorn345** [comment]
No no no. Lol I just look at my map from my phone. The localhost:5000 is on my laptop and it's not pulling. Maybe a new google maps API key? I have no clue

**Comment by istornz** [comment]
@BladeBorn345 Do you always get the error ?

**Comment by cyan** [comment]
if you can't connect to your map even using from the computer itself, you have a problem with your map, check your settings again i would say, (i think the problem should in the config.ini or the --command lines)

**Comment by 3Trace** [comment]
can someone give me an example of the server name/server info that would work? Or direct me to instructions instead? I tried putting an IP address in the "Server" place and "pool-173-00-00-00.phlapa.fios.verizon.net" in the "Server name" place. And a few variations of that but I can't seem to get it to work.

**Comment by BladeBorn345** [comment]
I did finally get past this issue. My IP address keeps changing randomly so I just need to check that before using. Can someone please explain to me how to update my ipokego though? I keep missing spawns and having them pop with only minutes left on them. Earlier a Dragonite at 0:57. I cried a little inside. Hahaha and I see there's now an IV finder added???
